extension/js/common/browser.ts

```diff
@@ -1266,6 +1266,7 @@ export class AttUI {
         return;
       }
       this.attachedFiles[uploadFileId] = newFile;
+      onSuccess!().catch(Catch.reportErr);
```

Comment:
You will get exception if `onSuccess` was not defined (see other note, adding callback may not be necessary).